src/handlers/prioritize.rs

```diff
@@ -38,6 +38,18 @@ impl Handler for PrioritizeHandler {
                     if e.label.as_ref().expect("label").name == config.label {
                         // We need to take the exact same action in this case.
                         return Ok(Some(Prioritize::Start));
+                    } else {
+                        if let Ok(glob) = glob::Pattern::new(&config.priority_labels) {
+                            let issue_labels = event.issue().unwrap().labels();
+                            let label_name = &e.label.as_ref().expect("label").name;
+
+                            if !issue_labels.iter().any(|l| glob.matches(&l.name))
+                                && config.prioritize_on.iter().any(|l| l == label_name)
+                            {
+                                // We need to take the exact same action in this case.
+                                return Ok(Some(Prioritize::Start));
```

Comment:
Please use `Prioritize::Label` here. It will label the issue with `I-prioritize`, thus sending a message on Zulip (but only if the issues has not `I-prioritize` yet)

Reply:
Done!